nextroute/schema/fleet.go

```diff
@@ -121,60 +127,38 @@ type Limits struct {
 	Duration string `json:"duration,omitempty"`
 }
 
+// dynamicDefaultKey is used when only scalar quantities / capacities are given.
+const dynamicDefaultKey = "default"
+
 // ToNextRoute converters a legacy cloud fleet input into nextroute input format.
 func (fleetInput FleetInput) ToNextRoute() (Input, error) {
 	input := Input{
-		Defaults: &Defaults{},
-	}
-	stopCompats := make([]string, 0)
-	vehicleCompats := make([]string, 0)
-	// Use default values and add special handling for CompatibilityAttributes
-	// and HardWindows.
-	if fleetInput.Defaults != nil && fleetInput.Defaults.Stops != nil {
-		input.Defaults.Stops = &StopDefaults{
-			UnplannedPenalty:        fleetInput.Defaults.Stops.UnassignedPenalty,
-			Quantity:                fleetInput.Defaults.Stops.Quantity,
-			MaxWait:                 fleetInput.Defaults.Stops.MaxWait,
-			Duration:                fleetInput.Defaults.Stops.StopDuration,
-			TargetArrivalTime:       fleetInput.Defaults.Stops.TargetTime,
-			EarlyArrivalTimePenalty: fleetInput.Defaults.Stops.EarlinessPenalty,
-			LateArrivalTimePenalty:  fleetInput.Defaults.Stops.LatenessPenalty,
-		}
-		if fleetInput.Defaults.Stops.CompatibilityAttributes != nil {
-			stopCompats = *fleetInput.Defaults.Stops.CompatibilityAttributes
-		}
-		if fleetInput.Defaults.Stops.HardWindow != nil {
-			input.Defaults.Stops.StartTimeWindow = *fleetInput.Defaults.Stops.HardWindow
-		}
+		//Defaults: &Defaults{},
 	}
+	// stopCompats := make([]string, 0)
```

Comment:
Why all this dead code (same applies to all dead code in this file)

Reply:
sorry, was not expecting you to look so quickly :D fixed just now with another push.